### ISSUE TITLE
A11y_Code.visualstudio.com_Visual studio code_Top navigation bar_High contrast: In high contrast black mode , search Image and Placeholder text "search Docs" is overlapped with the search box. 

### ISSUE THREAD

**Issue by MukallaDivya** [microsoft/vscode-docs#5150]
## GitHub Tags:
#A11yTCS;#Win10;#WCAG4.3.1;#DesktopWeb;#A11ySev2;#ChromiumEdge;#P1-VisualStudioCodeServices-Web-Feb2022;#P1_Visual Studio Code Services;#Visual Studio Code Services;
## Environmental Details:
#URL: https://code.visualstudio.com/License
## OS & Browser Details :
Windows 10 Enterprise 21H2(OS Build 19044,1466)
Microsoft Edge Version 98.0.1108.43 (Official build) (64-bit)
## Pre-Requisite:
To apply high contrast follow the below steps:
1. Open settings, tab till Ease of access and press enter.
2. Tab till High contrast press enter.
3.  From the drop down, select a proper theme by pressing enter. Use one theme i.e. 'black' .
4. Tab till apply button and press enter.

## Repro Steps :
1. Launch the URL https://code.visualstudio.com/License.
2. Turn on High contrast in settings.
3. Observe the "Search Box" , whether that placeholder text "Search Docs" and the "search Image" is properly presented or not.

## Actual Result :
 In high contrast , search Image and Placeholder text "search Docs" is overlapped with the search box. 
## Expected Result:
Search Image and Placeholder text "search Docs"   should not be overlapped with the search box.
## User Impact :
Users not able to understand clearly and got confuse, when that "search box" received  the focus.
## Attachments :
![Bug_5150_high contrast_controls are getting Overlapped](https://user-images.githubusercontent.com/99255594/153129551-18a30c8d-8e63-4ad1-ae23-04e5aab4e266.jpeg)

**Comment by rzhao271** [comment]
Wow it's pretty off.
![Screenshot showing incorrect searchbox outline](https://user-images.githubusercontent.com/7199958/153070668-6fec5fe8-a4a6-4c82-bf8b-b8bb55616dc7.PNG)

It appears like that on all pages, too.

**Comment by MukallaDivya** [comment]
Check again this bug in the below environment.
URL : https://code.visualstudio.com/License
 
Here, we can see that search image and the placeholder text "search docs" is not overlapped with the search box.
so, the issue is fixed.
![Screenshot (130)](https://user-images.githubusercontent.com/99255594/157237482-22289148-0a57-42eb-aa3c-bb0b393982f7.png)
